packages/@lwc/babel-plugin-component/src/decorators/wire/transform.ts

```diff
@@ -38,7 +38,7 @@ function getWiredParams(
         .filter((property) => isObservedProperty(property as NodePath<types.ObjectProperty>))
         .map((path) => {
             // Need to clone deep the observed property to remove the param prefix
-            const clonedProperty = t.cloneDeep(path.node) as types.ObjectProperty;
+            const clonedProperty = t.cloneNode(path.node) as types.ObjectProperty;
```

Comment:
@nolanlawson This change is not directly related to the actual issue. It only addresses a deprecation warning.

Reply:
I typically would prefer for this to be a separate PR, but no biggie. There is also one on line 289 btw.